### ISSUE TITLE
Berto Cloud driver breaks EA navigator

### ISSUE THREAD

**Issue by spike411** [davidmassot/Berto#9]
Hi,
I've identified a serious issue with your Berto cloud driver (version 7*, modified 14/10/2017 21:10*, downloaded on or around 22/5/2019).

The driver is very invasive, modifies configuration on the Linux OS level installing itself into `/etc/crontabs/root`, running shell scripts and tampering with the controllers networking (iptables).

This provably breaks network for the Android navigator app running inside the LXC container on EA controllers. About 10-15 minutes after the EA controller reboots the Android system loses the ability to connect anywhere outside the controller. That means IP cameras don't work, weather data does not work, cover art from internet services such as TuneIn or Deezer doesn't etc. Anything that requires the navigator itself to fetch data over the network.

I've verified this on two systems, on OS 2.10.5, 2.10.6 and 3.0.0.

I like your drivers, but if you could, please provide a clean, non-invasive version of the Cloud driver without any *backdoor-ish* shenanigans and don't do anything outside of the driver sandbox.

*The version/modified values apparently don't mean much, as I've downloaded the most recent version of the driver from your repository and the version/modified values remain the same even though some changes were arguably made to the driver.

![image](https://user-images.githubusercontent.com/118018/59241619-b709a380-8c08-11e9-9928-eab2d23c7a39.png)

**Comment by davidmassot** [comment]
Lukas,

The last release made several changes from the public version of the driver that made modifications to the base OS , I may have missed the crontab entry which is used by the Watchdog timer.

There are no 'backdoor-ish shenanigans' as you put. Originally all my drivers were only used by my customers which I support. When I created the public release most of the extras were removed after discussions with Control4 themselves, the Watchdog timer should have been disabled as well, I will check that later, this periodically checks the 'open files' used by the director process.

I'm interested in the issue you're experience however as we have many installs running touchscreens, EA1 as navigators, EA controllers and HC setups and have not had any reports of problems with the touchscreens not connecting beyond the local network, if they need to, even VLAN setups.

Thanks

David
